source/routing/raptor.cpp

```diff
@@ -87,6 +87,7 @@ bool RAPTOR::apply_vj_extension(const Visitor& v,
             auto sp = st.stop_point;
             const auto sp_idx = SpIdx(*sp);
 
+            if (! valid_stop_points[sp_idx.val]) { continue; }
```

Comment:
It would be nice to test (maybe it's already the case somewhere) that it works fine with pickup dropoff forbidden

Reply:
10 lines up